### ISSUE TITLE
Implement theme: Watercolor (Professional)

### ISSUE THREAD

**Issue by rozniak** [rozniak/xfce-winxp-tc#13]
Issue to track implementation progress for the Watercolor (Professional) theme.

**Comment by rozniak** [comment]
For a little break whilst I'm doing #40 I thought I'd have a little poke around doing the XFWM4 portion of this theme. I'm quite chuffed with the results!

![image](https://user-images.githubusercontent.com/13258281/123558905-cd074400-d790-11eb-8b4b-74d001ade8a9.png)
![image](https://user-images.githubusercontent.com/13258281/123558914-d98b9c80-d790-11eb-991b-89917cc43cfb.png)

Will push this work into another branch later on - will be at `feat-13`.

**Comment by rozniak** [comment]
`feat-13` branch now exists.

**Comment by rozniak** [comment]
Theme tidied and merged into `feat-themes` awaiting further work.

**Comment by rozniak** [comment]
Working on this again in `feat-13`, this time working alongside the refactored theme code. Will hopefully mean once this theme is done there is a good transpile process for constructing themes from a common base.

**Comment by rozniak** [comment]
I have quite a nice looking Start button, though the height may depend on the XFCE panel stuff once I check overall accuracy. I have some work to do to properly fix the ability to theme XFCE panel's borders before checking.
![image](https://user-images.githubusercontent.com/13258281/159187863-6ad0b3a0-4531-477c-9630-0ff626574034.png)

**Comment by rozniak** [comment]
Back on the case for this one.

My plan is to actually sort out the Windows Classic theme so that it becomes the 'default' (ie. if you transpile the gtk-3-base as-is). That way Luna (Blue) can be properly separated and the theme structure will be more akin to what it actually is on Windows, including the colour scheme.

**Comment by rozniak** [comment]
Updating this issue, have gotten quite far with this, here's a few screenshots. Though there's obviously still plenty left to do.

Professional / Watercolor widget factory
![image](https://user-images.githubusercontent.com/13258281/188335538-16b67729-d5a5-44e4-8b15-6617643220bd.png)

Taskbars (Adwaita/Win Classic/Professional/Luna)
![image](https://user-images.githubusercontent.com/13258281/188335526-72350ac8-2f07-4a40-a874-e9caed381053.png)

Start menu Windows Classic
![image](https://user-images.githubusercontent.com/13258281/188335590-37803dba-e0d3-4334-adf6-58016c7f028e.png)

Dunno if anyone reads individual issue updates but since there hasn't been a merge to `master` in some time I figured I would share in progress stuff that is coming.

**Comment by rozniak** [comment]
Luna Start menu migrated over to the new parts system
![image](https://user-images.githubusercontent.com/13258281/190645966-114ea972-1228-418d-93fc-5f4a52f98be1.png)

Windows Classic Start menu pretty much sorted
![image](https://user-images.githubusercontent.com/13258281/190646052-a4fcd805-4b79-4eee-9515-e28c33ca93a4.png)

In progress Professional/Watercolor Start menu
![image](https://user-images.githubusercontent.com/13258281/190646148-1bb8cbb6-6bdf-41b5-8453-581a5e00ed59.png)
